components/iotserver-mgt/org.wso2.carbon.device.mgt.iot.common/src/main/java/org/wso2/carbon/device/mgt/iot/common/transport/mqtt/MqttConfig.java

```diff
@@ -0,0 +1,55 @@
+package org.wso2.carbon.device.mgt.iot.common.transport.mqtt;
+
+import org.wso2.carbon.device.mgt.iot.common.config.server.DeviceCloudConfigManager;
+import org.wso2.carbon.device.mgt.iot.common.config.server.datasource.ControlQueue;
+
```

Comment:
Would be good to have a class level comment explaining what this class contains.